tests/test_no_github_in_gitlab_generated_project.py

```diff
@@ -0,0 +1,66 @@
+from cookiecutter.main import cookiecutter
+from hamcrest import assert_that, empty, is_
+
+def test_no_github_in_gitlab_generated_project(tmp_path):
+    project_title = "Generated Python Project GitLab"
+    project_slug = project_title.lower().replace(" ", "_").replace("-", "_")
+
+    cookiecutter(
+        ".",
+        extra_context={"project_name": project_title},
+        no_input=True,
+        default_config=True,
+        overwrite_if_exists=True,
+        output_dir=tmp_path,
+    )
+
+    project_path = tmp_path / project_slug
+
+    files = project_path.glob("**/*")
+
+    files_containing_github = []
+    files_to_exclude = [
+        ".DS_Store",
+        ".flake8",
+        ".pre-commit-config.yaml",
+        "mypy.ini",
+        "scripts/install_pyenv.sh",
+        "scripts/install_poetry.sh",
+        "behave4cmd0/README.txt",
+        "tests/test_generated_python_project_gitlab.py",
+    ]
+    content_to_exclude = [
+        "github.com/pyenv/",
+        "githubusercontent.com/sdispater/",
+        "github.com/python-poetry/",
+        "github.com/lumengxi/",
+        "github.com/thejohnfreeman/",
+        "github.com/python/",
+        "github.com/psf/",
+        "github.com/audreyr/",
+        "github.com/opinionated-digital-center/python-library-project-generator",
+    ]
+
+    for file in files:
+        if str(file.relative_to(project_path)) in files_to_exclude or not file.is_file():
+            continue
+        with file.open() as f:
+            failed = False
+            lines = f.readlines()
+            for line in lines:
+                count = line.lower().count("github")
+                if count > 0:
+                    if count > 1:
+                        words = line.split(" ")
+                        for word in words:
+                            if not any([excluded_content in word for excluded_content in content_to_exclude]):
```

Comment:
Just before applying the count, maybe remoe all occurences of `content to exclude` words so there will be no duplication of this line

Reply:
Done